### PR TITLE
Separate about contact section

### LINKED ISSUE #386: Separate About and contact section
**Describe the bug**
On the about page the `About` and  `Contact` section are very close together. 
Almost look as if it's one sentence

**To Reproduce**
1. Go to https://igboapi.com/about

**Expected behavior**
The two sections should have enough space between them to easily tell them apart

**Screenshots**
Screen Shot 2021-06-23 at 16 32 13

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version 11.2.3

### PR DESCRIPTION
**Pull request by inezabonte**
closes #386 

### Before
Screen Shot 2021-07-06 at 14 10 06

### After
Screen Shot 2021-07-06 at 14 09 45

### CODE REVIEW

**Comment by inezabonte** [comment]
@ijemmao kindly review this.
The reviewers section is disabled on my side (only works for added contributors)

File: .gitignore [hunk 1]
```diff
@@ -139,3 +139,6 @@ dist
 # Backup directories
 ./igbo_api_db
 ./aws_pronunciations
+
+# misc
```
Comment (ijemmao) [inline .gitignore:143]: Why add a misc section? We're these files getting added in your commits?

**Review by ijemmao — COMMENTED** [review]
This looks really good! Can you confirm that this works on mobile too?

File: .gitignore (hunk 1 above)
Comment (inezabonte) [reply .gitignore:143]: Yes the file is present in the project. It is usually ignored.
Screen Shot 2021-07-06 at 15 40 48

**Comment by inezabonte** [comment]
> This looks really good! Can you confirm that this works on mobile too?

Yes it works on mobile
Screen Shot 2021-07-06 at 15 43 34

**Review by ijemmao — APPROVED** [review]
looks good! just need to see why's failing the tests!

**Pull request closed without merging**